src/main/java/pl/simpleascoding/tutoringplatform/advertisement/AdvertisementService.java

```diff
@@ -0,0 +1,14 @@
+package pl.simpleascoding.tutoringplatform.advertisement;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import pl.simpleascoding.tutoringplatform.advertisement.dto.Advertisement;
+import pl.simpleascoding.tutoringplatform.advertisement.dto.CreateAdvertisement;
+import pl.simpleascoding.tutoringplatform.util.rscp.RscpDTO;
+
+public interface AdvertisementService {
```

Comment:
same here. why is mapper private, and other stuff public?

Reply:
we are focusing on hide all stuff that doesn't require access from outside features. In this case you're commenting service which needs to be public because whole app communication will be based on services mostly.

Reply:
@MateuszCiapala ye, that's my mistake